scripts/analysis_0/nbconverted/2A_correct_variation_added.py

```diff
@@ -182,6 +183,9 @@
 # In[9]:
 
 
+# log10 number of experiments
+lst_num_experiments = np.log(lst_num_experiments)
```

Comment:
This is misleading. Don't name transformed variables the same...

Reply:
Also I am very confused by what these experiments are. There are 108 experiment ids listed above, what are all these `experiments` here?

Reply:
Very good question and something that is indeed to confusing to those who didn't actually do the analysis:)  For analysis 0, a simulated experiment is being defined as a random collection of samples.  So experiment 1 = the full compendium of simulated samples in 1 partition, experiment 2 = a compendium of samples partitioned into 2 experiments where each experiment had a different amount of random noise added to it.  

So these "simulated experiments" are not using these "experiment ids" that exist in the original data.  This comes in in analysis 1.  

I will add a comment to clarify this.

Reply:
In the future, consider naming them as what they are...

experiment (simulated experiments) = simulated_samples | simulated_experiments
experiment (experiment_ids) = original_samples | original_experiments | experiments

or something of that nature

Reply:
Thank you for the suggestion, I will try to clean this up in a future PR